doc/tech/README.md

```diff
@@ -0,0 +1,148 @@
+
+This nature of this disaggregation is what gives the para-virtualized interfaces the flexibility that makes them favourable in a virtualized environment.
+
+The downside of para-virtualization interfaces is the involvement of the hypervisor which may introduce latency and jitter that can impact the performance.
+
+<p align="center"><img src="./figures/tech_virtio.png" alt="virtio" title="Document Types" width="50%"/></p>
+<p align="center"><b>Figure 4:</b> Para-Virtualized interface components (software only).</p>
+
+<a name="4.1.2"></a>
+#### 4.1.2 Direct assignment with Virtual Technology.
+
+Direct assignment (or pass-through) is when an IO device is directly assigned to a workload by-passing the hypervisor. Direct assignment is supported in Intel platforms (using VT-d technology) and by AMD platforms (using AMD-V technology) as shown in **Figure 5**.
+
+Once an IO device is directly assigned to a workload, that workload will then have an exclusive access to that device and no other entities (including the hypervisor) can access it.
+
+This method provides better performance than the para-virtualized one as no hypervisor is involved but provides less flexibility and less portability.
+
+Having an IO device directly assigned to a workload means that the workload needs to run vendor specific drivers and libraries to be able to access that device which makes the workload less portable and dependent on a specific hardware type from a specific vendor which is not aligned with the overall strategy and goals of CNTT and hence this method of IO Virtualization should not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
+
+<p align="center"><img src="./figures/tech_vtd.png" alt="virtio" title="Document Types" width="50%"/></p>
+<p align="center"><b>Figure 5:</b> Direct Assignment with Virtual Technology.</p>
+
+<a name="4.1.3"></a>
+#### 4.1.3 Device Sharing with SR-IOV & Virtual Technology.
+
+Unlike Direct assignment, this method allows a hardware device to be shared amongst many workloads where each workload has an exclusive access to one region of the device (known as VF). 
+
+For this method to be possible, the IO device need to support Single Root Input Output Virtualization (SR-IOV) which allows it to present itself as multiple devices, known as Physical Functions, PFs, and Virtual Functions, VFs as presented in **Figure 6**.
+
+Each of those Virtual Functions can then be independently assigned exclusively to a workload (with the appropriate support by VT-d or AMD-V from the infrastructure).
+
+Similar to the previous method ("Direct Assignment"), this method provides better performance than the para-virtualized one but lacks the flexibility and the portability sought and therefore should also not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
```

Comment:
should - Sounds like a loophole. I'd prefer "must".
Consider "... this method provides better performance than para-virtualization, but lacks ..."
Would like to see a reference to the plan (i.e. where does the reader find it?).

Reply:
resolved. will add the plan to the document once the other PR for the transitional plan is merged.